admin/test/test_release.py

```diff
@@ -1612,6 +1708,120 @@ def test_branch_created_from_base(self):
         self.assertIn((u'NEW_FILE', 0), self.repo.index.entries)
 
 
+class CreatePipIndexTests(SynchronousTestCase):
+    """
+    Tests for :func:`create_pip_index`.
+    """
+    def setUp(self):
+        self.scratch_directory = FilePath(self.mktemp())
+        self.scratch_directory.makedirs()
+
+    def test_index_created(self):
+        """
+        A pip index file is created for all wheel files.
+        """
+        index = create_pip_index(
+            scratch_directory=self.scratch_directory,
+            packages=[
+                'Flocker-0.3.0-py2-none-any.whl',
+                'Flocker-0.3.1-py2-none-any.whl'
+            ]
+        )
+
+        expected = dedent(
+            "<!--This is an index for pip-->\n"
+            '<a href="Flocker-0.3.0-py2-none-any.whl">Flocker-0.3.0-py2-none-any.whl</a><br/>\n'  # noqa
+            '<a href="Flocker-0.3.1-py2-none-any.whl">Flocker-0.3.1-py2-none-any.whl</a><br/>\n'  # noqa
+        )
+        self.assertEqual(expected, index.getContent())
+
+    def test_only_wheels_included(self):
+        """
+        The pip index file created only references wheel files.
```

Comment:
This should at least also reference the sdists. But I'm not sure there is a good reason to exclude anything other than `index`.